pyvista/plotting/qt_plotting.py

```diff
@@ -432,7 +432,7 @@ def __init__(self, parent=None, title=None, off_screen=None,
 
         if rcParams["depth_peeling"]["enabled"]:
             for renderer in self.renderers:
-                self.enable_depth_peeling()
```

Comment:
Good catch. Can you make sure to update this branch with `master`. Also, note that depth peeling needs `self.ren_win.AlphaBitPlanesOn()`... is this done here and for the `Plotter` class?

`AlphaBitPlanesOn` was forgotten in #529 and might have something to do with the issues you had in https://github.com/pyvista/pyvista/pull/529#issuecomment-574076243

Reply:
same needs to be done for `Plotter`

Reply:
I guess we you could make a call from `self` first and depending on the `result` iterate through the renderers? What do you think?

Reply:
Or port all this in `BasePlotter` actually since it would be the same for `Plotter`